includes/Core/Util/Debug_Data.php

```diff
@@ -214,7 +224,7 @@ private function get_amp_mode_field() {
 	 */
 	private function get_site_status_field() {
 		$is_connected = $this->authentication->credentials()->has();
-		$using_proxy  = $this->authentication->get_oauth_client()->using_proxy();
+		$using_proxy  = $this->credentials->using_proxy();
```

Comment:
This line should reference the same `Credentials` instance as the line above 😄

Reply:
Fixed